packages/devtools_app/lib/src/standalone_ui/vs_code/devices.dart

```diff
@@ -50,18 +48,29 @@ class Devices extends StatelessWidget {
     );
   }
 
-  TableRow _createDeviceRow(VsCodeDevice device, {required bool isSelected}) {
+  TableRow _createDeviceRow(
+    ColorScheme colorScheme,
+    VsCodeDevice device, {
+    required bool isSelected,
+  }) {
     return TableRow(
+      decoration: BoxDecoration(color: isSelected ? colorScheme.primary : null),
       children: [
-        Align(
-          alignment: Alignment.centerLeft,
+        SizedBox(
+          width: double.infinity,
           child: TextButton(
-            child: Text(device.name),
+            style: TextButton.styleFrom(
+              alignment: Alignment.centerLeft,
+              shape: const ContinuousRectangleBorder(),
+            ),
+            child: Text(
+              device.name,
+              style:
+                  TextStyle(color: isSelected ? colorScheme.onPrimary : null),
```

Comment:
can we do `Theme.of(context).regularTextStyle.copyWith(color: ...)`?

Reply:
Done! I also applied regularTextStyle to the `TextButton` because it was making the text bold by default which didn't match the debug sessions.

![image](https://github.com/flutter/devtools/assets/1078012/d971e647-bc45-491d-8389-c2365d83ffc7)